airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/templates/connector_registry_locations.html

```diff
@@ -1,12 +1,16 @@
 {% extends 'base.html' %}
 
-{% block title%}Connector Registries{% endblock %}
+{% block title%}Airbyte Connector Registry{% endblock %}
 
 {% block content %}
-<h1>Connector Registries</h1>
+<h1>Airbyte Connector Registries</h1>
 <h2>Sources</h2>
-{{ sources_table }}
+<div class="table-container test">
+  {{ sources_table }}
```

Comment:
I think we should pass a list of dict here and generate the table from this list of dict with a for loop in the template.